tests/usage.txt

```diff
@@ -1,25 +1,17 @@
-cargo-sweep.exe-sweep 
+Usage: cargo-sweep sweep [OPTIONS] <--stamp|--file|--time <TIME>|--installed|--toolchains <TOOLCHAINS>|--maxsize <MAXSIZE_MB>> [PATH]
 
-USAGE:
-    cargo-sweep.exe sweep [FLAGS] [OPTIONS] <--stamp|--file|--time <days>|--installed|--toolchains <toolchains>|--maxsize <maxsize>> [path]
+Arguments:
+  [PATH]  Path to check
 
-FLAGS:
-    -d, --dry-run      Dry run which will not delete any files
-    -f, --file         Load timestamp file in the given path, cleaning everything older
-    -h, --help         Prints help information
-        --hidden       The `recursive` flag defaults to ignoring directories that start with a `.`, `.git` for example
-                       is unlikely to include a Cargo project, this flag changes it to look in them.
-    -i, --installed    Keep only artifacts made by Toolchains currently installed by rustup
-    -r, --recursive    Apply on all projects below the given path
-    -s, --stamp        Store timestamp file at the given path, is used by file option
-    -V, --version      Prints version information
-    -v, --verbose      Turn verbose information on
-
-OPTIONS:
-        --maxsize <maxsize>          Remove oldest artifacts until the target directory is below the specified size in
-                                     MB
-    -t, --time <days>                Number of days backwards to keep
-        --toolchains <toolchains>    Toolchains (currently installed by rustup) that should have their artifacts kept.
-
-ARGS:
-    <path>    Path to check
+Options:
+  -d, --dry-run                  Dry run which will not delete any files
+  -f, --file                     Load timestamp file in the given path, cleaning everything older
+      --hidden                   The `recursive` flag defaults to ignoring directories that start with a `.`, `.git` for example is unlikely to include a Cargo project, this flag changes it to look in them
+  -i, --installed                Keep only artifacts made by Toolchains currently installed by rustup
+  -m, --maxsize <MAXSIZE_MB>     Remove oldest artifacts until the target directory is below the specified size in MB
```

Comment:
looks like `-m` is new as an alias - seems fine, just wanted to call out the new feature.

Reply:
Oops, I didn't notice it. Added to the PR description :sweat_smile:.